src/types/params/dealer.ts

```diff
@@ -0,0 +1,19 @@
+import { PaginationParams } from "./index"
+import { DealerSortParams } from "../sort"
+
+export interface DealerLocationFilter {
+  cityId?: string
+  radius?: string
+}
+
+export interface DealerQueryParams {
+  feature: string[]
```

Comment:
You could use a feature array https://github.com/carforyou/carforyou-api-client-pkg/blob/master/src/types/models/product.ts#L1 `Feature[]`

Reply:
hmmm not really sure about that. need to check that

Reply:
done